gobblin-runtime/src/main/java/gobblin/runtime/task/TaskUtils.java

```diff
@@ -26,7 +26,7 @@
  */
 public class TaskUtils {
 
-  private static final String TASK_FACTORY_CLASS = "gobblin.runtime.taskFactoryClass";
+  public static final String TASK_FACTORY_CLASS = "gobblin.runtime.taskFactoryClass";
```

Comment:
The reason this is private is because it gives us the freedom to change names or the way we determine task class in the future, without worrying that someone is using this key directly.